symbolic-debuginfo/src/elf.rs

```diff
@@ -214,14 +217,16 @@ impl<'data> ElfObject<'data> {
 
     /// Determines whether this object exposes a public symbol table.
     pub fn has_symbols(&self) -> bool {
-        !self.elf.syms.is_empty()
+        !self.elf.syms.is_empty() && !self.elf.dynsyms.is_empty()
```

Comment:
```suggestion
        !self.elf.syms.is_empty() || !self.elf.dynsyms.is_empty()
```

Easy to miss, but this failed the testsuite:
```
   12       │-        has_symbols: true,
         12 │+        has_symbols: false,
```

Reply:
Ah yes, silly mistake on my part